implementations/rust/ockam/ockam_command/tests/bats/local/reset.bats

```diff
@@ -34,6 +34,9 @@ teardown() {
   # reset the OCKAM_HOME directory
   run_success "$OCKAM" reset --yes
 
+  # it may, or may not exist, so we remove it to have a consistent state
+  run_success rm -f "$OCKAM_HOME"/application_database.sqlite3-wal
```

Comment:
Should this be done as part of the `reset` command?

Reply:
The application database is untouched by the ockam reset, and at the end of the reset the application database `wal` file may or may not exist.
When it exists, the `ls` would have a different output, breaking the assertion.
The alternative would be to assert either outputs with or without the `wal` file.